### ISSUE TITLE
Github releases page shows bogus date for upcoming 1.8.0 release

### ISSUE THREAD

**Issue by banshee** [tlaplus/tlaplus#668]
The text on the https://github.com/tlaplus/tlaplus/releases page says that the 1.8.0 release was from Dec 6, 2020:

    The Clarke release
    @lemmy lemmy released this on Dec 6, 2020

I'm assuming that's incorrect, since 1.7.1 was released on Dec 31, 2020?

**Comment by lemmy** [comment]
Yes, 1.8.0 has not been released.  The date (Dec 6, 2020) is when we created the draft release, whose artifacts we have been updating ever since.

**Comment by lemmy** [comment]
Feel free to reopen.
